bin/kano-ui-autostart

```diff
@@ -40,28 +40,30 @@ function track_startup
     fi
 }
 
-# Is this the first time we executed this boot?
-test -f /run/lock/desktop_init_run
-DESKTOP_RUN=$?
-touch /run/lock/desktop_init_run
-FIRST_BOOT=1
 
-if [ "$DESKTOP_RUN" -eq 1 ] ; then
+# Detect if we are running for the first time since boot
+boot_control_file=/var/run/lock/kano_ui_autostart_first_boot
+if [ -f $boot_control_file ]; then
+    FIRST_BOOT=0
+else
+    touch $boot_control_file
+    FIRST_BOOT=1
+fi
```

Comment:
@skarbat, it looks to me like this is conflating two of the variables in the old version:
DESKTOP_RUN, which was true if this was the first run *since booting*
FIRST_BOOT, which was true if this was the first boot *since this account was created*

We still need FIRST_BOOT because we want do do some housekeeping after account creation. But this change initialises it in the way that DESKTOP_RUN was before, using a file in `/var/run/lock/` which is deleted every boot. So now FIRST_BOOT activities will be run every boot.
(`/var/run/lock/` is a tmpfs, so it won't be preserved across boots)